src/map/mob.cpp

```diff
@@ -2904,8 +2900,8 @@ int mob_dead(struct mob_data *md, struct block_list *src, int type)
 		if( mapdrops != nullptr ){
 			// Process map wide drops
 			for( const auto& it : mapdrops->globals ){
-				if( rnd_chance( it.second->rate, 10000 ) ){
-					mob_item_drop( md, dlist, mob_setdropitem( it.second.get(), 1, md->mob_id ), 0, it.second->rate, homkillonly || merckillonly );
+				if( rnd_value( 0, 100000 ) < it.second->rate ){
+					mob_item_drop( md, dlist, mob_setdropitem( it.second.get(), 1, md->mob_id ), 0, it.second->rate, (homkillonly || merckillonly), true );
```

Comment:
I would prefer multiplying by 10 at these two calls and not touch the other function at all.
The mob_item_drop function is already unreadable enough with all those short ifs... :-(